src/cpp/session/session-options.json

```diff
@@ -19,6 +19,26 @@
       ]
    },
    "options": {
+      "automation": [
+         {
+            "name": {"constant": "kRunAutomationSessionOption", "value": "run-automation"},
+            "isHidden": true,
+            "memberName": "runAutomation_",
+            "type": "bool",
+            "defaultValue": false,
+            "implicitValue": true,
+            "description": "Run automation tests and exit."
+         },
+         {
+            "name": {"constant": "kAutomationAgentSessionOption", "value": "automation-agent"},
```

Comment:
I wonder how we should set this when connecting to an RStudio Server instance? Do we need a separate option to run RStudio Server in automation mode?